src/commands/utils/pr.ts

```diff
@@ -0,0 +1,71 @@
+import { request } from '@octokit/request';
+import { Colors } from 'discord.js';
+import { CommandDefinition } from '../../lib/command';
+import { CommandCategory, Roles } from '../../constants';
+import { makeEmbed } from '../../lib/embed';
+
+const syntaxHelp = '\nSyntax:\nA32NX repo: `.pr <id>`\nAny FBW repo: `.pr <repo> <id>`';
+const permittedRoles = [
+    Roles.ADMIN_TEAM,
+    Roles.MEDIA_TEAM,
+    Roles.MODERATION_TEAM,
+    Roles.DEVELOPMENT_TEAM,
+    Roles.BOT_DEVELOPER,
+    Roles.CONTRIBUTOR,
+    Roles.QA_TRAINEE,
+    Roles.QA_TESTER,
+];
+
+const noQueryEmbed = makeEmbed({
+    title: 'PR Error | Missing Query',
+    description: `Invalid command!${syntaxHelp}`,
+    color: Colors.Red,
+});
+
+const invalidEmbed = makeEmbed({
+    title: 'PR Error | Invalid',
+    description: `Something went wrong! Did you provide the correct repo/PR id?${syntaxHelp}`,
+    color: Colors.Red,
+});
+
+const noPermEmbed = makeEmbed({
+    title: 'PR | Permission missing',
+    description: 'You do not have permission to use this command.',
+    color: Colors.Red,
+});
+
+export const pr: CommandDefinition = {
+    name: 'pr',
+    description: `Retrieves the link of the provided GitHub PR.${syntaxHelp}`,
+    category: CommandCategory.UTILS,
+    executor: async (msg) => {
+        if (!msg.member.roles.cache.some((role) => permittedRoles.map((current) => current.toString()).includes(role.id))) {
+            return msg.reply({ embeds: [noPermEmbed] });
+        }
+
+        const command = msg.content.replace('.', '').split(/ +/);
+
+        if (/\.pr +\d+/i.test(msg.content)) {
```

Comment:
```suggestion
        if (/\.pr +#?\d+/i.test(msg.content)) {
```
This adds support for the original `.pr #1` and `.pr 1` (and any amount of spaces between)